# Chapter 7

In [4]:
from math import sqrt

### Survey Sampling & Inference

A statistician must use accuracy and precision to measure the quality of a survey. One must describe the important features of sampling distribution, including the standard error (std).


### The Central Limit Theorem

This theorem is used to estimate proportions in a population. It tells us that, if some basic conditions are met, the sampling distribution of the sample proportion is close to the Normal distribution.

If we take a random sample from a population, and if the sample size is large and the population size is much larger, the sampling distribution of p^ is approximately Normal with a mean of p and a std of:

SE = sqrt(p(1-p) / n)

#### Conditions for CLT

1) Sampling is random and independant
2) Large sample: The sample size, n , is large enough that the sample expects at least 10 successes and 10 failures
3) Big population: If sampling is done without replacement, the population must be at least 10x larger than the sample size.

np^ >= 10 AND n(1-p^) >= 10

#### Example: Vegetarians

According to statisticbrain.com, 59% of vegetarians are women. Suppose a random sample of 500 vegetarians is taken. What is the approx probability that the proportion of 

Step 1) We must check conditions: 

- The sample was random and independant
- There is a large sample size: 500(.59) = 295, and 295 > 10
- We can assert that there are at least 10(500) = 5000 vegetarians in the population.
- CLT has been met, therefore the distribution should be approximatly normal:

In [12]:
SE= sqrt((0.59*0.41)/500)
round(SE,4)

0.022

To find the probability that the population in the sample is greater than 63%, we can use technology to find the probability of getting a value larger than 0.63 in a N(0.59, 0.022) distribution.

According to calculations, there is only a 3.45% chance of receiving those results again. Based on that, we can assume those results are unlikely.

### Key Idea

The Central Limit Theorem tells us whether or not the study is presenting accurate results, allowing us to determine if the data can accuratley represent a population of people. 